src/voicecall.c

```diff
@@ -117,8 +117,8 @@ struct emulator_status {
 };
 
 static const char *default_en_list[] = { "911", "112", NULL };
-static const char *default_en_list_no_sim[] = { "119", "118", "999", "110",
-						"08", "000", NULL };
+static const char *default_en_list_no_sim[] = { "000", "08", "112", "110",
+    "118", "119", "120", "122", "911", "999",  NULL };
```

Comment:
112 and 911 should not be added to the list, as they are already in default_en_list, which is added always.